runtime/vm/vmprops.c

```diff
@@ -370,6 +381,7 @@ addPropertiesForOptionWithAssignArg(J9JavaVM *vm, const char *optionName, UDATA
 		UDATA indexLen = 1;
 
 		do {
+			/* The allocated return value for optionArg needs to be freed after use */
```

Comment:
There are 4 places calling this, why only comment one of them?

Reply:
Already added the function comment for `getOptionArg`.